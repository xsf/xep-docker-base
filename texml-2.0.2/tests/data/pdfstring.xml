<TeXML>
  _/0189:_@ABYZ[_`abyz{_&#x44f;
  <spec cat="nl"/>
  <pdf>_/0189:_@ABYZ[_`abyz{_&#x44f;</pdf>
  <spec cat="nl"/>
  _/0189:_@ABYZ[_`abyz{_&#x44f;
</TeXML>
