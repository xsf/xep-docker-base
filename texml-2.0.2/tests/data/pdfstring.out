\_/0189:\_@ABYZ[\_`abyz\{\_\cyrchar\cyrya{}
\000\137\000\057\000\060\000\061\000\070\000\071\000\072\000\137\000\100\000\101\000\102\000\131\000\132\000\133\000\137\000\140\000\141\000\142\000\171\000\172\000\173\000\137\004\117

\_/0189:\_@ABYZ[\_`abyz\{\_\cyrchar\cyrya{}
